src/content/docs/usage/ide.mdx

```diff
@@ -332,7 +332,7 @@ Daytona allows you to connect to your Workspace using JetBrains WebStorm IDE.
 Daytona allows you to connect to your Workspace using Cursor IDE.
 
 :::note
-Install [Cursor](https://www.cursor.com/) and ensure it's in your `PATH`.
+Install [Cursor](https://www.cursor.com/) and ensure it is in your `PATH`.
 
 After installing the Cursor IDE, run the `Install 'cursor' command` from the command palette.
```

Comment:
would be more specific: After installing the Cursor IDE, open Cursor and run the `Install 'cursor' command` from the command palette.